SITCOM-1690: Update data products section for CCR1 (#26)
lsst-sitcom/sitcomtn-005: dm.tex

	\item  "Should be done to enter Operations; but waiver likely to be granted if not met," i.e., we could enter Operations without this fulfilled, for first 3 years.
	\item  "Overall capability/efficiency/ease of use/etc., may be reduced but science will not critically suffer if not done." Could enter operations without this requirement fulfilled, and have the soperations team decide whether they want to pursue it.
\end{enumerate}

The verification status of requirements in the DMSR will be reported at each of the Construction Closeout Reviews.

> Priority 1a requirements are expected to be fully verified by CCR1, coinciding with the start of on-sky commissioning.

Reviewer: We are working on this but we won't get them all done by CCR1.

> Priority 1a requirements are expected to be fully verified by CCR1, coinciding with the start of on-sky commissioning.

Reviewer: ```suggestion
Most priority 1a requirements are expected to be fully verified by CCR1, coinciding with the start of on-sky commissioning with ComCam. Those that are not will be verified by the end of ComCam on sky commissioning and ready for the star of commissioning with LSSTCam.
```

Priority 1b requirements are expected to be fully verified by CCR3 to demonstrate readiness for the handover to Operations.

> Most of the priority 2 requirements will be verified by CCR4 in the course of demonstrating the capability to produce, document, and serve scientifically validated survey-scale data products to the science community via the Early Science Program \citeds{rtn-011}.

Reviewer: No - some will be but I'm not expecting to make substantial progress between CCR3 and CCR4. CCR4 is a formality.

> Most of the priority 2 requirements will be verified by CCR4 in the course of demonstrating the capability to produce, document, and serve scientifically validated survey-scale data products to the science community via the Early Science Program \citeds{rtn-011}.

Reviewer: ```suggestion
Most priority 2 requirements are expected to be verified during the course of on sky commissioning and early operations. For those that are not, a waiver will be sought  to enter operations and they will be completed within the first three years of operations
```

\end{itemize}

\subsubsection{Criteria for Completeness}
The criteria for completeness are described in \ref{sec:dm-completeness}.

> The project team shall process the data from the one (or more) of the Science Validation Surveys to produce a Data Release and make it available to the Commissioning Team through the DM Science User Interface as well as a subset for the EPO Public User Interface.

Reviewer: ```suggestion
The project team shall process the data from the one (or more) of the Science Validation Surveys to produce a Data Preview and make it available to the Commissioning Team through the Rubin Science Platform as well as a subset for the EPO Public User Interface.
```

\subsubsection{Pre-Operations Interactions}
None, unless there are non-compliance issues against the DMSR requirements and specifications.

\subsubsection{Artifacts for Completion}
The high-level artifacts for completion of the DRP pipelines are detailed in \ref{sec:dm-artifacts}.

%%%%%% RSP %%%%%%%%%%%
\subsection{Rubin Science Platform}
% {\it Responsible: Gregory Dubois-Felsmann (and Leanne Guy)}

\subsubsection{Objectives}

> The objectives of this Operational Requirement are to ensure that the Rubin Science Platform (RSP), including the DM Science User Interface, have been verified against requirements, and that the LSST science community can access, visualize, interact with, and analyze LSST data products.

Reviewer: ```suggestion
The objectives of this Operational Requirement are to ensure that the Rubin Science Platform (RSP) has been verified against requirements, and that the LSST science community can access, visualize, interact with, and analyze LSST data products.
```
